website/docs/feature-flag-tutorials/use-cases/a-b-testing.md

```diff
@@ -29,84 +31,123 @@ The feature flag we are creating is considered an ‘Experimentation’ flag typ
+![Create a feature flag by filling out the form fields.](/img/use-case-create-experiment-flag.png)
 
 Once you have completed the form, you can click ‘Create feature flag’.
 
-Your new feature flag has been created and is ready to be used. Upon returning to your projects view, enable the flag for your development environment, which makes it accessible to use in your app.
+Your new feature flag has been created and is ready to be used. Next, we will configure the A/B testing strategy for your new flag.
 
-![Enable the development environment for your feature flag for use in your application.](/img/tutorial-enable-dev-env.png)
+### Target Users for A/B Testing
 
-Next, we will configure the A/B testing strategy for your new flag.
+With an A/B testing strategy, you’ll be able to:
 
-### Implementing a Default Activation Strategy for A/B Testing
+-   Determine the percentage of users exposed to the new feature
+-   Determine the percentage of users that get exposed to each version of the feature
 
-An important Unleash concept that enables developers to perform an A/B test is an [activation strategy](/reference/activation-strategies). An activation strategy defines who will be exposed to a particular flag or flags. Unleash comes pre-configured with multiple activation strategies that let you enable a feature only for a specified audience, depending on the parameters under which you would like to release a feature.
+To target users accordingly, we will create an [activation strategy](/reference/activation-strategies). This Unleash concept defines who will be exposed to a particular flag or flags. Unleash comes pre-configured with multiple activation strategies that let you enable a feature only for a specified audience, depending on the parameters under which you would like to release a feature.
 
 ![Anatomy of an activation strategy](/img/anatomy-of-unleash-strategy.png)
 
-Different strategies use different parameters. Predefined strategies are bundled with Unleash. The default strategy is the gradual rollout strategy with 100% rollout, which basically means that the feature is enabled for all users. In this case, we have only enabled the flag in the development environment for all users in the previous section.
+Different strategies use different parameters. Predefined strategies are bundled with Unleash. The default strategy is the gradual rollout strategy with 100% rollout, which basically means that the feature is enabled for all users. In this tutorial, we will adjust the percentage of users who have access to the feature.
 
+:::note
 Activation strategies are defined on the server. For server-side SDKs, activation strategy implementation is done on the client side. For frontend SDKs, the feature is calculated on the server side.
+:::
+
+On the feature flag view in Unleash, click on the ‘Add your first strategy’ button to navigate to a (default) gradual rollout strategy form.
```

Comment:
You no longer need to do this, you can just enable the feature flag in the environment. But if you want still explain it the same way:

```suggestion
Open your feature flag and click **Add strategy**.
```